client/src/components/Attachment/UploadAttachment.js

```diff
@@ -10,6 +10,7 @@ import React, { useState } from "react"
 import { toast } from "react-toastify"
 import Settings from "settings"
 import { RELATED_OBJECT_TYPE_TO_ENTITY_TYPE } from "utils_links"
+import utils from "utils"
```

Comment:
Imports are not properly sorted; please run `yarn run prettier:format` before pushing your changes…
https://github.com/NCI-Agency/anet/actions/runs/5952125491/job/16143384235?pr=4464#step:5:21